tests/test.py

```diff
@@ -1,4 +1,18 @@
 from pprint import pprint
-from trav import Travis
+import pytest
```

Comment:
You should probably follow PEP guidelines. import pytest goes to the top, followed by pprint and then trav.